modules/swagger-codegen/src/main/java/io/swagger/codegen/DefaultCodegen.java

```diff
@@ -1481,7 +1481,10 @@ public CodegenProperty fromProperty(String name, Property p) {
         property.title = p.getTitle();
         property.getter = "get" + getterAndSetterCapitalize(name);
         property.setter = "set" + getterAndSetterCapitalize(name);
-        property.example = toExampleValue(p);
+        String example = toExampleValue(p);
```

Comment:
Instead of change the logic in this function (which may affect things elsewhere) just don't set a property value if the example is the string `null`.